turbiniactl

```diff
@@ -321,7 +333,8 @@ if __name__ == '__main__':
       print request.to_json().encode('utf-8')
     else:
       log.info(
-          u'Creating PubSub request with evidence {0:s}'.format(evidence_.name))
+          u'Creating PubSub request {0:s} with evidence {1:s}'.format(
```

Comment:
We are mixing u'' and '' in this file. Can we stick with one?

Reply:
Fixed.